packages/crayons-core/src/components.d.ts

```diff
@@ -6,6 +6,7 @@
  */
 import { HTMLStencilElement, JSXBase } from "@stencil/core/internal";
 import { AccordionToggleEvent } from "./components/accordion/accordion";
+import { CountryCode } from "libphonenumber-js/types";
```

Comment:
Please order the imports. move this import one line above so that relative imports are grouped together.

Reply:
@arvindanta : After changing the line import positions, When the transpile finished it's resetting as above.